src/loaders/data-sources/AnnDataSource.test.js

```diff
@@ -10,19 +10,19 @@ describe('sources/AnnDataSource', () => {
     expect(zGroup.zarr_format).toEqual(2);
   });
 
-  it('loadCellSetIds returns ids for location in store', async () => {
+  it('loadObsVariables returns ids for location in store', async () => {
     const dataSource = new AnnDataSource({
       url: 'http://127.0.0.1:8080/anndata/anndata-dense.zarr',
     });
-    const ids = await dataSource.loadCellSetIds(['obs/leiden']);
+    const ids = await dataSource.loadObsVariables(['obs/leiden']);
```

Comment:
Ah, I see now why `obs` and `var`... If we had `loadCellNames()` and `loadGeneNames` then this could be `loadCellSetIds(['leiden'])` had have `loadCellSetIds` add the `obs/` prefix.

Reply:
Since `AnnDataSource.loadObsIndex()` loads the `obs` (implicitly), should we have the full path of `obs/leiden` in `AnnDataSource.loadObsVariables()`? 

```javascript
const ids = await dataSource.loadObsVariables(['obs/leiden']);
// vs
const ids = await dataSource.loadObsVariables(['leiden']);
```

Reply:
I think it would be harder/less intuitive to remove `obs/` from all the incoming paths since they are all specified as `obs/path`.

Reply:
Got it, so these are paths defined in the `options` in the vitessce schema? (just so I understand)

Reply:
Yup, exactly.  Perhaps in the future, we could not do that (so cell information has to come form `obs` for example) but for now it's full paths (relative the AnnData store's path)